downstream/assemblies/hub/assembly-validated-content.adoc

```diff
@@ -22,55 +22,42 @@ Possible values are `certified` or `validated`.
 If missing both content sets will be uploaded.
 |====
 
-== {Valid}
+== Installing validated content using the tarball
+
+If you are not using the bundle installer, a standalone tarball, `ansible-validated-content-bundle-1.tar.gz` can be used instead.
+The standalone tarball can be used to update validated contents later in any environment, when a newer tarball becomes available and without having to re-run the bundle installer.
+
+To obtain the tarball, navigate to https://access.redhat.com/downloads/content/480 and select *Ansible Validated Content*.
+
+You require the following variables to run the playbook. 
+
+[cols="20%,50%",options="header"]
+|====
+| Name | Description 
+| *`automationhub_admin_password`* | Your administration password.
+| *`automation_hub_api_token`* | The API token generated for your {HubName}.
+| *`automationhub_main_url`* | For example, `\https://automationhub.example.com`
+| *`automation_hub_require_content_approval`* | Boolean (`true` or `false`)
+
+This must match the value used during {HubName} deployment.
```

Comment:
Can you also mention this variable is set to `true` by default in the installer so people don't need to go check?